pyproject.toml

```
@@ -299,3 +279,119 @@ files = [
 [tool.docc.output]
 path = "docs"
 extension = ".html"
+
+[tool.vulture]
+exclude = ["tests/fixtures/", "eest_tests/"]
+ignore_names = ["pytest_*"]
+
+[tool.ruff]
+exclude = [
+    '^\.cache/',
+    '^\.git/',
+    '^\.pytest_cache/',
+    '^\.ruff_cache/',
+    '^\.tox/',
+    '^\.venv/',
+    '^\.vscode/',
+    'tests/fixtures/*',
+    'eest_tests/*',
+    'vulture_whitelist.py',
+]
```

Apparently these are supposed to be [globset](https://docs.rs/globset/latest/globset/#syntax)s. So this should be something like:

```suggestion
exclude = [
    ".cache/",
    ".git/",
    ".pytest_cache/",
    ".ruff_cache/",
    ".tox/",
    ".venv/",
    ".vscode/",
    "tests/fixtures/*",
    "eest_tests/*",
    "vulture_whitelist.py",
]
```